src/components/animation/Animation.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { AnimationWrapper, TextMessages, TextBubble } from './styledAnimation'
+import Avatar from '../avatar/Avatar';
+import { AnimationData, leftAvatarConfig, rightAvatarConfig } from '../../constants/animationConst';
+import uniqueId from 'lodash.uniqueid';
+
+const Animation = () => {
+
+  return (
+    <AnimationWrapper>
+      <TextMessages>
+        <Avatar
+          config={leftAvatarConfig}
+        />
+        {
+          AnimationData.map(({ color, bgColor, align, text }, i) => {
+            return (
+              <TextBubble
```

Comment:
This is cool, I don't understand framer yet, but I'm getting a sense of it from the way this component is constructed.